frontend/src/components/explorer/MapViewer.vue

```diff
@@ -163,6 +163,16 @@ export default {
     await this.$store.dispatch('maps/getMapsListing', this.model);
     this.$store.dispatch('maps/initFromQueryParams', this.$route.query);
     this.loadMapFromParams();
+    const footer = document.getElementById('footer');
+    footer.querySelector('#footer-logos').style.display = 'none';
+    footer.querySelector('#copyright-standard').style.display = 'none';
+    footer.querySelector('#copyright-mapviewer').style.display = 'block';
```

Comment:
These lines should go as high up as possible, since at the moment the footer on the Map Viewer flickers - the users can see the layout change.